x-pack/legacy/plugins/apm/server/routes/create_api.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { merge } from 'lodash';
+import Boom from 'boom';
+import { InternalCoreSetup } from 'src/core/server';
+import { Request, ResponseToolkit } from 'hapi';
+import * as t from 'io-ts';
+import { PathReporter } from 'io-ts/lib/PathReporter';
+import {
+  ServerAPI,
+  RouteFactoryFn,
+  HttpMethod,
+  Route,
+  Params
+} from './typings';
+import { debugRt } from './default_api_types';
+
+export function createApi() {
```

Comment:
Can you create a test for this?

Reply:
Yep! Will also do that for clientApmApi.